### PR TITLE
p2p: mconn track last message for pongs

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

File: internal/p2p/conn/connection.go [hunk 1]
```diff
@@ -516,11 +521,9 @@ FOR_LOOP:
 				// never block
 			}
 		case *tmp2p.Packet_PacketPong:
-			select {
-			case c.pongTimeoutCh <- false:
-			default:
-				// never block
-			}
+			// do nothing, we updated the "last message
+			// recieved" timestamp above, so we can ignore
```
Comment (creachadair) [inline internal/p2p/conn/connection.go]:
```suggestion
			// received" timestamp above, so we can ignore
```

**Review by creachadair — APPROVED** [review]
A couple optional things, but LGTM.

File: internal/p2p/conn/connection.go [hunk 2]
```diff
@@ -370,6 +369,10 @@ FOR_LOOP:
 			break FOR_LOOP
 		case <-c.quitSendRoutine:
 			break FOR_LOOP
+		case <-pongTimeout.C:
+			// the point of the pong timer is to check to
+			// see if we'e
```
Comment (creachadair) [inline internal/p2p/conn/connection.go:374]: This comment appears to have lost its

**Review by creachadair — APPROVED** [review]

File: internal/p2p/conn/connection.go (hunk 2 above)
Comment (tychoish) [reply internal/p2p/conn/connection.go]: lo

**Pull request merged**